mmaction/datasets/pipelines/augmentations.py

```diff
@@ -403,10 +313,30 @@ def __call__(self, results):
         if img_w > self.size:
             x_offset = int(np.random.randint(0, img_w - self.size))
 
+        if 'crop_quadruple' not in results:
```

Comment:
What are `crop_quadruple`s used for

Reply:
Crop_quadruple is used to map the cropped image to the original image. Each crop_quadruple is a quadruple (x, y, w, h), each element in [0, 1], indicate the cropped image is mapped to which patch on the original image. The interface can be used for testing with advanced augmentations.